agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/PhenotypeAnnotationCurationIndexer.java

```diff
@@ -0,0 +1,470 @@
+package org.alliancegenome.indexer.indexers.curation;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.alliancegenome.api.entity.*;
+import org.alliancegenome.curation_api.model.entities.*;
+import org.alliancegenome.curation_api.model.entities.base.SubmittedObject;
+import org.alliancegenome.curation_api.model.entities.ontology.DOTerm;
+import org.alliancegenome.curation_api.model.entities.ontology.ECOTerm;
+import org.alliancegenome.curation_api.model.entities.ontology.OntologyTerm;
+import org.alliancegenome.es.util.ProcessDisplayHelper;
+import org.alliancegenome.indexer.RestConfig;
+import org.alliancegenome.indexer.config.IndexerConfig;
+import org.alliancegenome.indexer.indexers.Indexer;
+import org.alliancegenome.indexer.indexers.curation.service.AGMPhenotypeAnnotationService;
+import org.alliancegenome.indexer.indexers.curation.service.AllelePhenotypeAnnotationService;
+import org.alliancegenome.indexer.indexers.curation.service.GenePhenotypeAnnotationService;
+import org.alliancegenome.indexer.indexers.curation.service.VocabularyService;
+import org.alliancegenome.neo4j.entity.SpeciesType;
+import org.alliancegenome.neo4j.repository.DiseaseRepository;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.*;
+import java.util.Map.Entry;
+import java.util.concurrent.LinkedBlockingDeque;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.groupingBy;
+
+@Slf4j
+public class PhenotypeAnnotationCurationIndexer extends Indexer {
```

Comment:
There are a lot of methods in this class that are not being used and should be removed if that is the case.